saos-webapp/src/main/resources/message/common.properties

```diff
@@ -114,20 +115,23 @@ judgmentSearch.formField.commonCourt = Sąd
 judgmentSearch.formField.commonDivision = Wydział
 judgmentSearch.formField.supremeChamber = Izba sądu najwyższego
 judgmentSearch.formField.supremeChamberDivision = Wydział izby sądu najwyższego
-judgmentSearch.formField.chooseCommonCourt = Wybierz Sąd
-judgmentSearch.formField.chooseCcDivision = Wybierz wydział
-judgmentSearch.formField.chooseSupremeChamber = Wybierz izbę
-judgmentSearch.formField.chooseSupremeChamberDivision = Wybierz wydział izby
-judgmentSearch.formField.chooseScJudgmentForm = Wybierz formę orzeczenia
-judgmentSearch.formField.chooseScPersonnelType = Wybierz typ składu sędziowskiego
+judgmentSearch.formField.chooseCommonCourt = Dowolny sąd
+judgmentSearch.formField.chooseCcDivision = Dowolny wydział
+judgmentSearch.formField.chooseSupremeChamber = Dowolna izba
+judgmentSearch.formField.chooseScDivision = Dowolny wydział izby
+judgmentSearch.formField.chooseScJudgmentForm = Dowolna forma
+judgmentSearch.formField.chooseScPersonnelType = Dowolny typ
 judgmentSearch.tooltip.settings = Pokaż ustawienia wyszukiwania
 
+judgmentSearch.settings.iconAlt = Ikona ustawień wyszukiwania
```

Comment:
It's ok. Image alternative text should describe what is on the image, not what it does.